kedro-datasets/tests/conftest.py

```diff
@@ -89,6 +89,11 @@ def save_version(request):
     return request.param or generate_timestamp()
 
 
+@fixture(params=[None])
+def database(request):
+    return request.param
+
+
```

Comment:
```suggestion
@fixture(params=[None])
def database(request):
    return request.param


```
@mark-druffel Just for your future information (I will update this now), the fixtures under `kedro-datasets/tests/conftest.py` are shared across all tests, so we don't need to add an Ibis-specific fixture here.